awscli/customizations/s3/subcommands.py

```diff
@@ -266,12 +266,20 @@ def get_endpoint(service, region, endpoint_url, verify):
                                 verify=verify)
 
 
+def get_client(session, region, endpoint_url, verify):
```

Comment:
``CommandArchitecture`` object will use it as well, and that does not inherit from any command class.